.github/workflows/ci.yml

```diff
@@ -58,16 +56,33 @@ jobs:
       - run: python -m pip install -r requirements.txt
       - run: ${{ matrix.lint-command }}
 
-  readme:
+  build-and-validate:
     runs-on: ubuntu-latest
     steps:
-      - run: sudo apt install -y gettext
+      - run: |
+          sudo apt update
+          sudo apt install -y gettext
       - uses: actions/setup-python@v5.6.0
       - uses: actions/checkout@v4
-      - name: Install Python dependencies
-        run: python -m pip install --upgrade pip setuptools wheel twine readme-renderer
-      - run: python setup.py sdist bdist_wheel
-      - run: python -m twine check dist/*
+      - name: Install uv and readme-renderer
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install uv
+          uv venv
+          source .venv/bin/activate
+          uv pip install readme-renderer
+      - name: Clean previous builds
+        run: rm -rf build dist *.egg-info
+      - name: Compile translations
+        run: uv run compile-translations
+      - name: Build distributions
+        run: uv build
+      - name: Validate installability
+        run: uv pip install dist/*.whl
```

Comment:
What shouldn't be needed. Nothing can fail when installing a wheel, since it's compiled already.

Reply:
You can try the source install, which will fail ;)